compiler/vmgen.nim

```diff
@@ -2077,7 +2077,7 @@ proc toKey(s: PSym): string =
     result.add s.name.s
     if s.owner != nil:
       if sfFromGeneric in s.flags:
-        s = s.owner.owner
+        s = s.instantiatedFrom.owner
```

Comment:
Why not just `s.owner`?

Reply:
Because the module owner is the one that uses it, not the one that has it. The `proc` is still in the one that has it